src/test/java/au/com/centrumsystems/hudson/plugin/buildpipeline/functionaltest/BuildSecurityTest.java

```diff
@@ -38,6 +38,7 @@ public void init() throws Exception {
         jr.jenkins.rebuildDependencyGraph();
     }
 
+    @Ignore
```

Comment:
This whole Selenium-based test suite was failing and excluded by default, except during releases, except for these two tests. Ignoring these two tests like all the others.